runtime/lua/vim/filetype.lua

```diff
@@ -680,6 +698,16 @@ local extension = {
   bbl = "tex",
   latex = "tex",
   sty = "tex",
+  cls = function(path, bufnr)
+    local line = getline(bufnr, 1)
+    if line:sub(1, 1) == "%" then
```

Comment:
```suggestion
    if line:find("^%%") then
```

(I think we have decided on using `find` consistently -- sorry for my back-and-forth on this!)